tests/unit/config/config_test.py

```diff
@@ -1322,6 +1322,29 @@ def test_load_bind_mount_relative_path(self):
         assert mount.type == 'bind'
         assert mount.source == expected_source
 
+    def test_load_bind_mount_relative_path_with_tilda(self):
+        base_file = config.ConfigFile(
+            'base.yaml', {
+                'version': '3.4',
+                'services': {
+                    'web': {
+                        'image': 'busybox:latest',
+                        'volumes': [
+                            {'type': 'bind', 'source': '~/web', 'target': '/web'},
+                        ],
+                    },
+                },
+            },
+        )
+
+        details = config.ConfigDetails('.', [base_file])
+        config_data = config.load(details)
+        mount = config_data.services[0].get('volumes')[0]
+        assert mount.target == '/web'
+        assert mount.type == 'bind'
+        assert (not mount.source.startswith('~')
+                and mount.source.endswith('\\web'))
```

Comment:
That's going to fail on UNIX systems. Use `endswith('{}web'.format(os.path.sep))` instead?

Reply:
Your truth. Thanks.